samples/bootloader/prj.conf

```diff
@@ -1,12 +1,11 @@
 CONFIG_IS_SECURE_BOOTLOADER=y
-CONFIG_BOOT_BANNER=n # Need this to turn off PRINTK (the bootlader has its own)
+CONFIG_BOOT_BANNER=n
```

Comment:
Do we still need this?

Reply:
Hm, the boot banner is actually printed AFAICS

Reply:
I made the debug conf file the default. I can change it back to reduce confusion.

Reply:
my bad. Since the rest of zephyr is built with debug-by-default I say we keep it as is. 

BTW is it documented that there is a "hot-swap" option to go from debug mode to production mode?

Reply:
The problem is that you can't really enable and disable debug from Kconfig easily without side effects, hence the debug conf file. I don't know what the standard way to switch conf file is.

    cmake ... -DCONF_FILE=debug.conf

?